check-for-travis.rb

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env ruby
+require 'octokit'
+
+client = Octokit::Client.new :netrc => true
```

Comment:
better to use

```rb
client = Octokit::Client.new :access_token => ENV['GITHUBTEACHER_TOKEN'] || raise("You need a GitHub Teacher access token")
```

`ENV['GITHUBTEACHER_TOKEN']` is available on the heroku instance this is running on, e.g. https://github.com/githubteacher/practice-sinatra/blob/master/collaborators.rb#L61.